### ISSUE TITLE
kernel module built on OpenBSD is broken

### ISSUE THREAD

**Issue by james-d-mitchell** [semigroups/Semigroups#898]
See the following issue for details:

https://github.com/digraphs/Digraphs/issues/562

**Comment by james-d-mitchell** [comment]
See also: 

https://github.com/semigroups/Semigroups/issues/328

which seems to be a similar issue.

**Comment by dimpase** [comment]
With https://github.com/digraphs/Digraphs/issues/562 fixed by @fingolfin - I suppose a similar fix is needed here.
Otherwise
```
Error, LOAD_DYN: failed to load kernel module /home/dima/tmp/gap/pkg/semigroups//bin/x86_64-unknown-openbsd7\
.1-default64-kv8/semigroups.so, Cannot load specified object in
  LOAD_DYN( filename ) at /home/dima/tmp/gap/lib/files.gd:595 called from 
<function "LoadDynamicModule">( <arguments> )
 called from read-eval loop at /home/dima/tmp/gap/pkg/semigroups/init.g:22
Error, was not in any namespace at /home/dima/tmp/gap/lib/variable.g:269 called from
LEAVE_NAMESPACE(  ); at /home/dima/tmp/gap/lib/package.gi:1327 called from
ReadPackage( pkgname, "init.g" ); at /home/dima/tmp/gap/lib/package.gi:1688 called from
<function "LoadPackage">( <arguments> )
 called from read-eval loop at *stdin*:1
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk> 
```

**Comment by dimpase** [comment]
fixed by #973

**Comment by fingolfin** [comment]
@dimpase is this resolved now, possibly except for what's discussed in #938?

**Comment by dimpase** [comment]
the #938 certainly needs to be resolved - e.g
 for SageMath installs.

Would you like me to fix it, or prefer to do it yourself?

**Comment by dimpase** [comment]
should be fixed by #954

**Comment by james-d-mitchell** [comment]
Resolved in v5.3.0
